epictrack-api/src/api/models/work_issue_updates.py

```diff
@@ -0,0 +1,39 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Model to handle all operations related to Issues."""
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from .base_model import BaseModelVersioned
+
+
+class WorkIssueUpdates(BaseModelVersioned):
+    """Model class for updates Connected to an issue."""
+
+    __tablename__ = 'work_issue_updates'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    description = Column(String(2000), nullable=False)
+
+    work_issue_id = Column(ForeignKey('work_issues.id'), nullable=False)
+    work_issue = relationship('WorkIssues', back_populates='updates')
+
+    def as_dict(self):  # pylint:disable=arguments-differ
```

Comment:
Even if you don't put this as_dict() method here, you still be able to get it from the BaseModel

Reply:
do we even need as_dict? since we use marshamallow , i wasnt sure..just kept it there..

Reply:
> do we even need as_dict? since we use marshamallow , i wasnt sure..just kept it there..

Sometimes I use it when I update stuff. like, status.update(status.as_dict(recursive=False)). I want to pass a dict. yes, I can dump a json using Marshmallow response. I haven't tried that. Lets check the possibility

Reply:
removed for now..